### PR TITLE
[EMCalCorrectionFramework] Add setters to the size of NxM clusterizer clusters

### PR DESCRIPTION
**Pull request by gconesab**

### CODE REVIEW

File: PWG/EMCAL/EMCALtasks/AliEmcalCorrectionClusterizer.h [hunk 1]
```diff
@@ -69,6 +69,8 @@ class AliEmcalCorrectionClusterizer : public AliEmcalCorrectionComponent {
   Bool_t                 fJustUnfold;                     ///< just unfold, do not recluster
   Float_t                fUnfoldCellMinE;                 ///< min energy cell threshold, after unfolding
   Float_t                fUnfoldCellMinEFrac;             ///< min fraction of cell energy after unfolding  
+  Int_t                  fNxMRowDiff;                     ///< NxN clusterizer, max number rows from center cell: 1->3, 2->5, 3->7
+  Int_t                  fNxMColDiff;                     ///> NxN clusterizer, max number columns from center cell:1->3, 2->5, 3->7
```
Comment (raymondEhlers) [inline PWG/EMCAL/EMCALtasks/AliEmcalCorrectionClusterizer.h:73]: Typo in the streaming comment - `>` -> `<`

**Comment by raymondEhlers** [comment]
LGTM - just a small typo

File: PWG/EMCAL/EMCALtasks/AliEmcalCorrectionClusterizer.h (hunk 1 above)
Comment (gconesab) [reply PWG/EMCAL/EMCALtasks/AliEmcalCorrectionClusterizer.h:73]: Do you think this is considered so far from the ///?
Anyway, I will change it.

Comment (raymondEhlers) [reply PWG/EMCAL/EMCALtasks/AliEmcalCorrectionClusterizer.h:73]: `///<` is meaningful, but `///>` isn't. The rest of the `>` in the comment are totally fine

**Comment by raymondEhlers** [comment]
Hi @gconesab , I couldn't highlight a particular character, so I think we miscommunicated here a bit, sorry! (posting again instead of on the comment because the comment may have gotten lost in the notifications)

In that line that I highlighted, the `1->3, 2->5, 3->7` in the comment is totally fine. It's that the streaming comment: you have `///>`, but it should be `///<`

**Comment by gconesab** [comment]
> Hi @gconesab , I couldn't highlight a particular character, so I think we miscommunicated here a bit, sorry! (posting again instead of on the comment because the comment may have gotten lost in the notifications)
> 
> In that line that I highlighted, the `1->3, 2->5, 3->7` in the comment is totally fine. It's that the streaming comment: you have `///>`, but it should be `///<`

Ah! thanks, well spotted, changed

**Comment by gconesab** [comment]
+1

**Pull request merged**